packages/cli/src/commands/init/init.js

```diff
@@ -26,6 +26,16 @@ type Options = {|
   npm?: boolean,
 |};
 
+function getProjectPath({
+  projectName,
+  customProjectPath = '',
+}: {
+  projectName: string,
+  customProjectPath: string,
+}) {
+  return path.join(customProjectPath, projectName);
```

Comment:
So this will create a `./custom-path/AppName`. I don't think this is correct approach. I think we should run in a similar manner as `git` command.

By default, `folder` is $projectName.

When you run `react-native init MyProject .`, this will init in CWD. When you run `react-native init MyProject ./custom-location`, this will init in `custom-location`, meaning that `package.json` will be available at `custom-location/package.json`.

Just like when you do git clone repoName.